### PR TITLE
Events tracking

### PR DESCRIPTION
**Pull request by vycius**
* Activities are automatically tracked by Firebase;
* Fragments are tracked  by `AnalyticsUtil::trackCurrentFragment`;
* As example for further tracking events included `AnalyticsUtil::trackViewProblem` which will give insights about problems clicked;

All these events can be seen in Firebase analytics.

For access to Firebase please write to me or @RenatBuga

### CODE REVIEW

File: app/src/main/java/lt/vilnius/tvarkau/utils/AnalyticsUtil.kt [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package lt.vilnius.tvarkau.utils
+
+import android.app.Activity
+import android.content.Context
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import com.google.firebase.analytics.FirebaseAnalytics
+import lt.vilnius.tvarkau.entity.Problem
+
+object AnalyticsUtil {
```
Comment (mjurkus) [inline app/src/main/java/lt/vilnius/tvarkau/utils/AnalyticsUtil.kt]: I would prefer if we could make it as an implementation.

`inferface Analytics`
`class RemoteAnalytics : Analytics`

and injected using dagger.
That way we could test implementation and integration. What do you think?

That one could write in fragments tests

`verify(analytics).trackProblem(any())`

**Review by mjurkus — COMMENTED** [review]
I'd like to discuss more about using injectable service rather than util class.

File: app/src/main/java/lt/vilnius/tvarkau/utils/AnalyticsUtil.kt (hunk 1 above)
Comment (vycius) [reply app/src/main/java/lt/vilnius/tvarkau/utils/AnalyticsUtil.kt]: It depends on our testing approaches, what do we want to test, where do we want to move with tests, how are we planning to develop the app. Let's move this discussion to https://github.com/vilnius/tvarkau-vilniu/pull/103

**Comment by ViliusKraujutis** [comment]
I'd say we merge this tracking improvement and refactor to injectables later. :+1:

**Review by mjurkus — APPROVED** [review]
Awesome!

**Pull request merged**